pkg/server/server.go

```diff
@@ -685,7 +687,7 @@ func (s *Server) Close() {
 	s.typesystemResolverStop()
 }
 
-func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequest) (*openfgav1.ListObjectsResponse, error) {
+func (s *Server) ListObjectsWithoutAuthz(ctx context.Context, req *openfgav1.ListObjectsRequest) (*openfgav1.ListObjectsResponse, error) {
```

Comment:
**Question: ** Why is this not a breaking change, or is it?

Reply:
This is an internal function and I will update it to make it non-public